public/admin-account.php

```diff
@@ -1,10 +1,26 @@
-<?php 
-  require_once("../utils/database.php");
-  require_once("../utils/authentication.php");
-  require_once("../utils/filter.php");
-  require_once("../utils/movies.php");
-  require_once("../utils/functions.php");
-  checkLoggedIn();
+<?php
+require_once("../utils/database.php");
+require_once("../utils/authentication.php");
+require_once("../utils/filter.php");
+require_once("../utils/movies.php");
+require_once("../utils/functions.php");
+checkLoggedIn();
+
+$error = '';
+
+if (isset($_GET['approve-user'])) {
+  $input["id"] = filterInputGet($_GET["approve-user"], "approve-user");
+
+  $user = getTableRecord('SELECT * FROM gebruiker WHERE id = ?', 'i', array($input['id']));
+  if (!isset($user['ERROR'])) {
+    if ($user['geverifieerd'] == 0) {
+      executeQuery('UPDATE gebruiker SET geverifieerd = ? WHERE id = ?', 'ii', array(1, $input['approve-user']));

```

Comment:
$input['approve-user'] bestaat niet, dit zou dan $input['id'] moeten zijn

Reply:
Zou je ook een terug koppeling kunnen geven aan de admin wanneer de gebruiker wel approved is?

Reply:
Dit gebeurt als alle data van de gebruiker word opgehaald. 

Wanneer die al is geapproved word die knop niet getoond